api/src/main/java/org/eclipse/microprofile/lra/annotation/LRA.java

```diff
@@ -184,20 +226,37 @@
     ChronoUnit timeUnit() default ChronoUnit.SECONDS;
 
     /**
+     * <p>
      * Normally if an LRA is present when a bean method is executed it will not
      * be ended when the method returns. To override this behaviour and force LRA
-     * termination on exit use the end element
+     * termination on exit use the end element.
+     * </p>
+     *
+     * <p>
+     * If the <code>end</code> value is set to <code>false</code>
+     * while the annotated method returns the {@link Response} status code
+     * with the meaning the LRA to be cancelled (see {@link #cancelOn()} and {@link #cancelOnFamily()})
+     * then this error state has precedence over the <code>end</code> attribute.
+     * It means the LRA will be cancelled despite the end was set to <code>false</code>.
+     * </p>
      *
      * @return true if an LRA that was present before method execution will be
      * terminated when the bean method finishes.
      */
     boolean end() default true;
 
     /**
+     * <p>
      * The cancelOnFamily element can be set to indicate which families of
-     * HTTP response codes will cause the current LRA to cancel. By default client
-     * errors (4xx codes) and server errors (5xx codes) will result in
-     * cancellation of the LRA.
+     * HTTP response codes will cause the current LRA to cancel.
+     * This expects the annotated method is the JAX-RS endpoint where the {@link Response}
+     * with the status code could be returned.
+     * </p>
+     *
+     * <p>
+     * By default client errors (<code>4xx codes</code>) and server errors (<code>5xx codes</code>)
```

Comment:
Will you also set this default on the element please.

Reply:
@mmusgrov I'm sorry but I'm not sure what `element` you mean. Would you be so kind to elaborate where I can add the info?

Reply:
The cancelOnFamily element in the annotation class LRA.java needs a default, ie you should add:

```java
Response.Status.Family[] cancelOnFamily() default {Response.Status.Family.CLIENT_ERROR, Response.Status.Family.SERVER_ERROR};
```

Reply:
good point. I added that there.